crates/re_renderer/src/view_builder.rs

```diff
@@ -423,25 +418,39 @@ impl ViewBuilder {
         Ok(self)
     }
 
+    fn draw_phase<'a>(
+        &'a self,
+        ctx: &'a RenderContext,
+        phase: DrawPhase,
+        pass: &mut wgpu::RenderPass<'a>,
+    ) -> Result<(), anyhow::Error> {
+        for queued_draw in &self.queued_draws {
+            if queued_draw.participated_phases.contains(&phase) {
+                (queued_draw.draw_func)(ctx, phase, pass, queued_draw.draw_data.as_ref())
+                    .context("drawing a view")?;
```

Comment:
this context could mention the phase, and maybe even the name of the `queued_draw`

Reply:
adding phase :). name is a bit too cumbersome now